vms/shift/utils.py

```diff
@@ -75,6 +75,27 @@ def create_volunteer_with_details(volunteer):
     return v1
 
 
+def create_volunteer_with_details_dynamic_password(volunteer):
+    """
+    Creates and returns volunteer with passed name and dates
+    """
+    u1 = User.objects.create_user(username=volunteer[0], password=volunteer[1])
+    v1 = Volunteer(
```

Comment:
inconsistency (minor though). Why not use Volunteer.objects.create() as you have used above (I know you have used a diff manager function, but the second one is a model instantiation via constructor which is different.

Reply:
The similar functions above have used this so I went with that.

Reply:
also, some general notes - please avoid using these kind of variables. `u1, a1, b2, c3'. it should be `user_1` or even better `user_a`, `user_b`, etc.

Reply:
Sure  :+1: will change it asap.